README.md

```diff
@@ -20,7 +20,7 @@ The main goal is to use modern sensors (mainly [I2C](https://github.com/orgua/iL
 - **DS2413 (0x3A) Dual channel addressable switch with input-sensing**
 - **DS2423 (0x1D) 4kbit RAM with Counter**
 - **DS2431 (0x2D) 1kbit protected EEPROM** (also known as DS1972 or DS28E07, same FC)
-- DS2432 (0x33) 1kbit protected EEPROM (basically a ds2431 with extra sha-engine)
+- **DS2432 (0x33) 1kbit protected EEPROM** (basically a ds2431 with extra sha-engine)
```

Comment:
you added no code to support the ds2432, so why you mark this device as feature complete? sha-engine is still completely missing